packages/clima_ui/lib/widgets/failure_snack_bar.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:clima_core/failure.dart';
+import 'package:flutter/material.dart';
+
+void showFailureSnackBar({
```

Comment:
This is not really a widget, so it doesn't belong in `widgets`. It would be more well-suited to `utils` as it was IMO.